src/consensus/aft/raft.h

```diff
@@ -1891,6 +1978,19 @@ namespace aft
 
       channels->send_authenticated(
         to, ccf::NodeMsgType::consensus_msg, response);
+
+      if (
+        consensus_type == ConsensusType::BFT &&
+        response.success == AppendEntriesResponseType::OK)
+      {
+        NodeCaughtUpMsg ncamsg = {
+          {raft_node_caught_up}, state->current_view, state->last_idx};
+        for (const auto& [node_id, _] : nodes)
+        {
+          channels->send_authenticated(
+            node_id, ccf::NodeMsgType::consensus_msg, ncamsg);
+        }
+      }
```

Comment:
should we be only sending this if the node is catching up?

Reply:
Good point, I think you're right, we only need this during catchup.